include/jemalloc/internal/ticker.h

```diff
@@ -150,26 +154,35 @@ ticker_geom_read(const ticker_geom_t *ticker) {
 JEMALLOC_NOINLINE
 #endif
 static bool
-ticker_geom_fixup(ticker_geom_t *ticker, uint64_t *prng_state) {
+ticker_geom_fixup(ticker_geom_t *ticker, uint64_t *prng_state,
+    bool delay_trigger) {
+	if (delay_trigger) {
+		ticker->tick = 0;
```

Comment:
With `tick` set to 0 here, the event will be triggered immediately after quitting the reentrancy. For cases where reentrancy occurs frequently in a sequential pattern, will the regression caused by extra events be a concern here?

Reply:
This is already down the event triggering path, so there won't be any extra events. But rather the event that was supposed to be triggered during reentrancy will be delay until the next normal invocation.